tests/installation/logs_from_installation_system.pm

```diff
@@ -36,6 +36,14 @@ sub run {
     my $dasd_path = get_var('DASD_PATH', '0.0.0150');
     select_console 'install-shell';
 
+    if (check_var('SYSTEM_ROLE', 'Common_Criteria') && is_sle) {
+        assert_script_run('mount /dev/vda2 /mnt');
```

Comment:
Lili,

Do we have to remount root file system here?

Reply:
BTW, I suggest you add more comments here why we need this change

Reply:
> Lili,
> 
> Do we have to remount root file system here?

yes, as /mnt does not exist after installation (await_install).

Reply:
> BTW, I suggest you add more comments here why we need this change

Added comments.